assets/multiqc_config.yml

```diff
@@ -3,9 +3,9 @@ custom_logo_url: https://github.com/nf-core/raredisease/
 custom_logo_title: "nf-core/raredisease"
 
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/raredisease/releases/tag/2.0.1" target="_blank">nf-core/raredisease</a>
+  This report has been generated by the <a href="https://github.com/nf-core/raredisease/tree/dev" target="_blank">nf-core/raredisease</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/raredisease/2.0.1/docs/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/raredisease/dev/docs/output" target="_blank">documentation</a>.
```

Comment:
if it's a release, it shouldn't point to dev.
Did you use `nf-core bump-version 2.1.0`?

Reply:
Apparently, I had not done that 🤦🏻 But its fixed now.